test/anyof.test.js

```diff
@@ -248,3 +248,34 @@ test('null value in schema', (t) => {
     t.fail()
   }
 })
+
+test('anyOf and $ref together', (t) => {
+  t.plan(0)
+
+  const schema = {
+    type: 'object',
+    properties: {
+      cs: {
+        anyOf: [
+          {
+            $ref: '#/definitions/Option'
+          },
+          {
+            type: 'boolean'
+          }
+        ]
+      }
+    },
+    definitions: {
+      Option: {
+        type: 'string'
+      }
+    }
+  }
+
+  try {
+    build(schema)
+  } catch (e) {
+    t.fail()
+  }
```

Comment:
Would you mind to also test if this serializes ok?

Reply:
I can only fix it for one level of nestedness: if one the anyOfs is a ref, it expands that in the generated code. But if one of them is another anyOf with a ref, then it blows up.

I couldn't figure out how to recursively expand schema

Reply:
I think one is better than nothing, and it might cater to the most common case. Would you mind throwing a proper error in that case something like "nested $ref are not supported yet".